app/src/main/java/com/sunlightlabs/android/congress/fragments/BillListFragment.java

```diff
@@ -446,7 +440,10 @@ private void longDate(TextView view, Date date) {
 		}
 		
 		private String actionText(Bill.Action action) {
-			return Utils.truncate(action.text, 80);
+            if (action.description != null)
+			    return Utils.truncate(action.description, 80);
+            else
+                return "";
```

Comment:
Gracefully handle the case where an action's description is missing (haven't seen this happen, to be clear).